tools/proteore_id_converter/id_converter.xml

```diff
@@ -0,0 +1,343 @@
+<tool id="IDconverter" name="ID Converter" version="2020.01.09">
```

Comment:
is that similar to this tool? https://github.com/bgruening/galaxytools/blob/dee6a6f4aa8cbee9f81692fc6f3a2e4ad1f3abf6/tools/general_mapping/general_mapping.xml